tests/postgres_tests/test_aggregates.py

```diff
@@ -428,6 +428,30 @@ def test_array_agg_distinct_true(self):
         values = AggregateTestModel.objects.aggregate(arrayagg=ArrayAgg('char_field', distinct=True))
         self.assertEqual(sorted(values['arrayagg']), ['Bar', 'Foo'])
 
+    def test_jsonb_agg_distinct_false(self):
+        values = AggregateTestModel.objects.aggregate(
+            jsonbagg=JSONBAgg(Func(
+                Value('char_field'), 'char_field',
+                function='jsonb_build_object'
+            ), distinct=False),
+        )
+        self.assertListEqual(
+            sorted(values['jsonbagg'], key=lambda x: x['char_field']),
+            [{'char_field': 'Bar'}, {'char_field': 'Foo'}, {'char_field': 'Foo'}]
+        )
+
+    def test_jsonb_agg_distinct_true(self):
+        values = AggregateTestModel.objects.aggregate(
+            jsonbagg=JSONBAgg(Func(
+                Value('char_field'), 'char_field',
+                function='jsonb_build_object'
```

Comment:
Do we need to use custom function? We could test the same using `JSONBAgg('char_field', distinct=True/False)`, e.g.
```python
    def test_jsonb_agg_distinct_false(self):
        values = AggregateTestModel.objects.aggregate(
            jsonbagg=JSONBAgg('char_field', distinct=False),
        )
        self.assertEqual(sorted(values['jsonbagg']), ['Bar', 'Foo', 'Foo'])
    
    def test_jsonb_agg_distinct_true(self):
        values = AggregateTestModel.objects.aggregate(
            jsonbagg=JSONBAgg('char_field', distinct=True),
        )
        self.assertEqual(sorted(values['jsonbagg']), ['Bar', 'Foo'])
```